src/services/impressions/dto.js

```diff
@@ -36,6 +36,10 @@ module.exports = {
           if (entry.label) keyImpression.label = entry.label;
           if (entry.bucketingKey) keyImpression.bucketingKey = entry.bucketingKey;
 
+          properties = {};
+          if (FS) properties.full_story_url = FS.getCurrentSessionURL();
```

Comment:
Running the current check I think it's enough. You are checking for a _truthy_ value. If it's falsey (undefined, false, null...) so it may be better.

Please state on that file that you are using a global variable called FS with the following comment on top of the file (after the license disclosure):
` /* global FS */ `
With that we avoid linters complaining about FS not being defined.